apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.ExecutableCommand.cs

```diff
@@ -223,6 +226,14 @@ private async Task<int> ExecuteDmlAsync(CancellationToken cancellationToken)
                 return checked((int) count);
             }
 
+            private async Task<SpannerDataReader> ExecuteDmlReaderAsync(bool enableGetSchemaTable, IDisposable resourceToClose, CancellationToken cancellationToken)
```

Comment:
```suggestion
            private async Task<SpannerDataReader> ExecuteDmlReaderAsync(CommandBehaviour behaviour, CancellationToken cancellationToken)
```
And calculate the others here, it's fine if that code is duplicated with ExceuteReaderAsync, its clearer to have the ExecuteDmlReaderAsync implementation completely separate from the ExecuteReaderAsync one. If they were sharing a lot more code that'd be different.